onnxruntime/test/providers/cpu/nn/pool_op_test.cc

```diff
@@ -1264,6 +1264,25 @@ TEST(PoolTest, LpPool) {
   test.Run();
 }
 
+TEST(PoolTest, LpPoolWithNegativeNumbers) {
+  OpTester test("LpPool");
+
+  test.AddAttribute("p", static_cast<int64_t>(1));
+  test.AddAttribute("auto_pad", "");
+  test.AddAttribute("strides", std::vector<int64_t>{2});
+  test.AddAttribute("pads", vector<int64_t>{0, 0});
+  test.AddAttribute("kernel_shape", vector<int64_t>{2});
+
+  std::vector<float> x_vals = {0.2f, -0.6f};
+  std::vector<int64_t> x_dims = {1, 1, 2};
+  std::vector<int64_t> expected_dims = {1, 1, 1};
+  std::vector<float> expected_vals = {0.4f};
```

Comment:
I think you want -0.4f and not 0.4f (sorry I didn't spot it either)

Reply:
done.